Modules/Utils/Sources/PocketCastsUtils/Extensions/UI/UITextView+scrollToRange.swift

```diff
@@ -4,34 +4,31 @@ import UIKit
 extension UITextView {
     public func scrollToRange(_ range: NSRange) {
         // Ensure layout is up-to-date
-        textStorage.addLayoutManager(layoutManager)
         layoutManager.ensureLayout(for: textContainer)
 
         let glyphRange = layoutManager.glyphRange(forCharacterRange: range, actualCharacterRange: nil)
 
-        layoutManager.enumerateEnclosingRects(forGlyphRange: glyphRange, withinSelectedGlyphRange: NSRange(location: NSNotFound, length: 0), in: textContainer) { [weak self] rect, _ in
-            guard let self else { return }
+        let rect = layoutManager.boundingRect(forGlyphRange: glyphRange, in: self.textContainer)
 
-            let finalRect = rect.offsetBy(dx: textContainerInset.left, dy: textContainerInset.top)
+        let finalRect = rect.offsetBy(dx: textContainerInset.left, dy: textContainerInset.top)
 
-            // Calculate the rectangle to scroll to such that the finalRect is centered
-            var visibleRect = CGRect(
-                x: finalRect.origin.x - (bounds.width / 2) + (finalRect.width / 2),
-                y: finalRect.origin.y - (bounds.height / 2) + (finalRect.height / 2),
-                width: bounds.width,
-                height: bounds.height
-            ).inset(by: contentInset)
-            visibleRect = .init(x: visibleRect.origin.x, y: visibleRect.origin.y + (contentInset.bottom / 2), width: visibleRect.width, height: visibleRect.height)
+        // Calculate the rectangle to scroll to such that the finalRect is centered
+        var visibleRect = CGRect(
+            x: finalRect.origin.x - (bounds.width / 2) + (finalRect.width / 2),
+            y: finalRect.origin.y - (bounds.height / 2) + (finalRect.height / 2),
+            width: bounds.width,
+            height: bounds.height
+        ).inset(by: contentInset)
+        visibleRect = .init(x: visibleRect.origin.x, y: visibleRect.origin.y + (contentInset.bottom / 2), width: visibleRect.width, height: visibleRect.height)
```

Comment:
Could this be simplified to `visibleRect.origin.y += contentInset.bottom / 2`?

Reply:
Done here: https://github.com/Automattic/pocket-casts-ios/pull/2155/commits/93985d1d4747a445ad557142c846a7bd5538de2e